turbo.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://turborepo.org/schema.json",
+    "globalDependencies": ["**/.env"],
+    "ui": "stream",
+    "tasks": {
+        "topo": {
+            "dependsOn": ["^topo"]
+        },
+        "build": {
+            "env": ["SUPABASE_ANON_KEY", "SUPABASE_API_KEY"],
```

Comment:
Is there a reason these are the only env variables included, excluding
VITE_MIXPANEL_TOKEN and VITE_ANTHROPIC_API_KEY ?